src/oc/bot/digest.clj

```diff
@@ -54,6 +55,28 @@
         attachments (map #(post-as-attachment daily (:name board) %) (:posts board))]
     (concat [(assoc (first attachments) :pretext (str "*" pretext "*"))] (rest attachments))))
 
+(defn- split-attachments
+  "Split message attachments into multiple message if over 16kb
+   https://api.slack.com/docs/rate-limits"
+  [attachments]
+  (let [default-split 5 ;; 4 posts plus header
+        four-split (partition default-split default-split nil attachments)
+        four-bytes (.getBytes (json/generate-string (first four-split) "UTF-8"))
+        four-count (count four-bytes)
+        bytes (.getBytes (json/generate-string attachments) "UTF-8")
+        byte-count (count bytes)
+        byte-limit 6000] ;; 16k is the limit but need to account for HTTP
+    (timbre/info "Slack limit?: " four-count byte-count byte-limit)
+    (if (> four-count byte-limit)
```

Comment:
@thepug can you add some comments about what's going on here?
Do you try to with 4 posts instead of 5 if the partition is not below 16kb?